client/my-sites/exporter/export-card/index.jsx

```diff
@@ -62,7 +63,12 @@ class ExportCard extends Component {
 						<div>
 							<h1 className="export-card__title">{ translate( 'Export your content' ) }</h1>
 							<h2 className="export-card__subtitle">
-								{ translate( 'Or select specific content items to export' ) }
+								{ config.isEnabled( 'export-media' )
+									? translate(
+											'Or select specific content items to export. Please note this does not include your media library, ' +
+												'but only text content like posts and pages.'
+									  )
+									: translate( 'Or select specific content items to export' ) }
```

Comment:
Along with removing the feature flag `export-media`, the description only has one variation now.

About:
> A better option could be to add a link to the text that will have the same effect than the chevron icon

If I'm not mistaken, the common practices for this when using `<FoldableCard>` is to set `clickableHeader` as `true` so that the whole card can be clicked to expand. I've also update the sentence as `Or click here to select specific content items to export` to emphasize that it can be clicked. 

Does that work for you? I'm happy to refine the approach, of course :)